cylc/rose/utilities.py

```diff
@@ -192,13 +192,27 @@ def identify_templating_section(config_node):
             "You should not define more than one templating section. "
             f"You defined:\n\t{'; '.join(defined_sections)}"
         )
-    elif 'jinja2:suite.rc' in defined_sections:
+    elif defined_sections:
+        return id_templating_section(list(defined_sections)[0])
+    else:
+        return id_templating_section('')
+
+
+def id_templating_section(
+    section: Optional[str] = None,
+    with_brackets: Optional[bool] = False
```

Comment:
Reminder that `Optional` just means "can be `None`"
```suggestion
    with_brackets: bool = False
```